### ISSUE TITLE
pre-training batch size mismatch between code and paper

### ISSUE THREAD

**Issue by youngwanLEE** [MCG-NJU/VideoMAE#64]
First of all, thanks for your excellent work.

I'm re-producing your work by using one node with 8 GPUs.

In your pre-training scripts for K400 and SSv2, 

the effective batch size : 32 x 8 nodes x 8 gpus = 2048 

but your paper denotes `1024`. 

Could you let me know which one is correct? 

Thanks in advance.

**Comment by yztongzhan** [comment]
Hi @youngwanLEE! The experiment of `batch size = 1024` (16 x 8 nodes x 8 gpus = 1024) can also yield favorable results.

**Comment by youngwanLEE** [comment]
@yztongzhan Thanks for your quick response!

Is it possible to let me know the exact reported accuracy?

Because I'm trying to use a batch size of `1024`, but the result is not better than yours. :(

**Comment by yztongzhan** [comment]
> Is it possible to let me know the exact reported accuracy?

We can achieve the results in our paper.

> I'm re-producing your work by using one node with 8 GPUs.

I **cannot** guarantee that the results can be reproduced with **8 GPUs** if you use some **tricks** to enable **8 GPUs** pre-training. In my previous trivial, I got 68.3% on SSV2 with 8 GPUs for both pre-training and fine-tuning (800 epochs). However, as shown in our MODEL_ZOO, we can achieve 69.6% on SSV2. Therefore, there exists a **gap** of about 1%.

**Comment by youngwanLEE** [comment]
@yztongzhan Thanks for your quick and kind reply :)

**Comment by youngwanLEE** [comment]
@yztongzhan 
Sorry to bother you, I have one more question.

If I set the effective batch size `1024`, can I use the learning rate of `1.5e-4` as in the paper?

**Comment by yztongzhan** [comment]
Hi @youngwanLEE! 
We set the **base** learning rate to 1.5e-4.
We do not have a concept of **effective** batch size.

**Comment by youngwanLEE** [comment]
@yztongzhan 
I mean the effective batch size as a total batch size. 
Anyway, Thanks for reply :)

**Comment by xxtars** [comment]
@yztongzhan 
Sorry to bother you, but I have one question. 
I don't have enough GPUs, so can I achieve a larger batch size by gradient accumulation? Would that be a problem? Does a smaller batch size make a drop in the final performance?
Thanks.
